src/app.js

```diff
@@ -119,8 +117,29 @@ function createApp (config, ledgers, backend, routeBuilder, routeBroadcaster, ro
   require('./services/auth')(passport, config)
 
   // Logger
-  koaApp.use(logger())
-  koaApp.use(errorHandler({log: log('error-handler')}))
+  koaApp.use(logger(log.create('koa')))
+  koaApp.use(logger.requestIdContext())
+
+  const isTrace = log.trace()
+  koaApp.use(logger.requestLogger({
+    updateRequestLogFields: function (fields) {
+      return {
+        headers: this.req.headers,
+        body: isTrace ? this.body : undefined,
+        query: this.query
+      }
+    },
+    updateResponseLogFields: function (fields) {
+      return {
+        duration: fields.duration,
+        status: this.status,
+        headers: this.headers,
+        body: isTrace ? this.body : undefined
+      }
+    }
+  }))
+  koaApp.use(errorHandler({log: log.create('error-handler')}))
+  koaApp.on('error', function () {})
```

Comment:
Otherwise, the app emits an error and the stack trace is logged directly to the console (it's already logged by the logger).

See: https://github.com/koajs/bunyan-logger#suppressing-default-error-stack-traces